game/graphic/bitmap.hh

```diff
@@ -54,3 +58,4 @@ void writePNG(fs::path const& filename, Bitmap const& bitmap, unsigned stride =
 void loadPNG(Bitmap& bitmap, fs::path const& filename);
 void loadJPEG(Bitmap& bitmap, fs::path const& filename);
 
+void load(Bitmap& bitmap, fs::path const& name);
```

Comment:
Hmmm would it be handy to be more explicit about visibility? I know methods/fields are private per default.. but i found this out only after googling.. It might be a bit more verbose but makes it very explicit. In c# we even get warnings when we don't specify a visibility modifier

Reply:
Sorry, I don't understand your point. I think the image loading code is very ugly and I would like to change it one day, but before I want to merge the branches which exists already.